src/languages/es.js

```diff
@@ -827,6 +827,9 @@ export default {
         privateDescription: 'Sólo las personas que están invitadas a esta sala pueden encontrarla',
         createRoom: 'Crea una sala de chat',
         roomAlreadyExists: 'Ya existe una sala con este nombre',
+        pleaseEnterRoomName: 'Por favor escribe el nombre de una sala',
+        pleaseChooseWorkspace: 'Por favor elige un espacio de trabajo',
+        pleaseChooseVisibility: 'Por favor elige una visibilidad',
```

Comment:
Remove this line.

Reply:
Which line do you mean?

Reply:
ohh you mean `pleaseChooseVisibility`. Okay sure.

Reply:
Yes, sorry for not being specific. The comment usually refers to the line it was left on.